ambry-tools/src/main/java/com/github/ambry/tools/perf/ServerReadPerformance.java

```diff
@@ -172,7 +173,7 @@ public static void main(String args[]) {
 
       ToolUtils.ensureOrExit(listOpt, options, parser);
 
-      long measurementIntervalNs = options.valueOf(measurementIntervalOpt) * SystemTime.NsPerSec;
+      long measurementIntervalNs = options.valueOf(measurementIntervalOpt) * TimeUnit.SECONDS.toNanos(1);
```

Comment:
i guess you could supply the value of the option directly to the timeunit converter instead of multiplying